README.md

```diff
@@ -44,27 +44,50 @@ where a person has a first name and a last name
 In a MVC pattern, a controller written in Javascript defines what a
 form does on submission. In technical terms, we call this _binding_.
 
-```
+```javascript
 // bind the view to the model
 $('#new-person-form').on('submit', function(){})
 ```
 
 This is great! This is exactly what we want a controller to
 do. It defines actions and doesn't mix itself up with our view.
-But what happens if we want to render a set of elements that
-share the same HTML and CSS?
 
-```
-// create new HTML elements based on the 
-// length of the array
-var people = [{...},{...}]
-var html = ''
-for (var i = 0; i < people.length; i++) {
-  var person = people[i]
-  html.append('<div class="person">...</div>')
-}
-// then write it directly to the page
-$('#people').html(html)
+But lets say we we want to display a table with an unknown 
```

Comment:
Love the feel the pain, then relieve the pain with Angular approach to exaplining